diffkemp/simpll/Config.cpp

```diff
@@ -93,8 +94,27 @@ Config::Config()
     }
     if (!VariableOpt.empty()) {
         // Parse --var option - find global variables with given name.
-        FirstVar = First->getGlobalVariable(VariableOpt, true);
-        SecondVar = Second->getGlobalVariable(VariableOpt, true);
+        std::string varName, varValue;
+        std::string vopt = VariableOpt;
+        size_t varOptDelimiterIndex = vopt.find(":");
+        if (varOptDelimiterIndex == std::string::npos)
+            varName = vopt;
+        else {
+            varName = vopt.substr(0, varOptDelimiterIndex);
+            varValue = vopt.substr(varOptDelimiterIndex + 1, vopt.size());
+        }
+
+        FirstVar = First->getGlobalVariable(varName, true);
+        SecondVar = Second->getGlobalVariable(varName, true);
+
+        if (vopt == varName) {
+            VarValue = nullptr;
+        } else if (varValue.empty() || varValue == "default")
+            UseDefaultValue = true;
```

Comment:
Wouldn't it be easier to directly pass the initializer to `VarValue`? It's true that you would need to have `VarValueFirst` and `VarValueSecond` but it seems somehow cleaner to me (and you could get rid of a block of code in `VarValueDependencySlicer`).